nemo/collections/asr/data/audio_to_text_dali.py

```diff
@@ -287,16 +312,14 @@ def __init__(
                 min_duration=min_duration,
                 max_duration=max_duration,
                 read_sample_rate=False,
-                read_text=True,
+                read_text=False,
+                read_idxs=True,
                 random_shuffle=shuffle,
                 shard_id=self.shard_id,
                 num_shards=self.num_shards,
                 pad_last_batch=False,
```

Comment:
I think you can switch it to true now - in 1.4 the bug in the `nemo_asr` was fixed so it should no longer crash. If `pad_last_batch` is set to False LastBatchPolicy.DROP or  LastBatchPolicy.PARTIAL discards useful samples from the next epoch.